Dockerfile

```diff
@@ -30,12 +30,20 @@ RUN make install
 # Get Hermes build
 FROM ghcr.io/informalsystems/hermes:1.4.1 AS hermes-builder
 
+# Get CometMock
+FROM informalofftermatt/cometmock:latest as cometmock-builder
+
+# Get GoRelayer
+FROM informalofftermatt/gorelayer:nogas AS gorelayer-builder
```

Comment:
Can you talk to @greg-szabo to get these dockerhub images transferred to an official Informal account?

Reply:
Also, it's actually very hard to see from these links what the actual version of CometMock used is. The only way seems to be to pull the container yourself, start it, go into it, go into the folder then check the git remote.

Reply:
Is there some way to do this with git without impacting build times etc?

Reply:
> Is there some way to do this with git without impacting build times etc?

Do you have something concrete in mind? This should not impact build times significantly - the two builder images are cached, the only operation that is repeated when code is changed is the copy, but this only copies relatively tiny binaries over.